### PR TITLE
Patch less with CVE-2022-46663

### PR DESCRIPTION
**Pull request by suresh-thelkar**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Patch less with CVE 2022 46663

###### Change Log  

- CVE-2022-46663

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-46663

###### Test Methodology

- Buddy Build
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=311053&view=results)

### CODE REVIEW

**Comment by suresh-thelkar** [comment]
Please note that It seems there are some infrastructure issue with ARM64 buddy builds and hence could not add it here.

File: SPECS/less/less.spec [hunk 1]
```diff
@@ -31,6 +32,10 @@ The Less package contains a text file viewer
 %{_mandir}/*/*
 
 %changelog
+* Fri Feb 17 2023 Suresh Thelkar <sthelkar@microsoft.com> - 590-2
+- Patch CVE-2022-46663
+- License verified.
```
Comment (mfrw) [inline SPECS/less/less.spec:37]: It is not needed as license is verified already on line 41
```suggestion
```

**Review by mfrw — APPROVED** [review]
LGTM :)

File: SPECS/less/less.spec (hunk 1 above)
Comment (suresh-thelkar) [reply SPECS/less/less.spec]: Thanks @mfrw for the review and approval. I have removed this statement as well in the latest commit.

**Comment by suresh-thelkar** [comment]
Seems Buddy build infrastructure issues are fixed for ARM64. I am able to run Buddy Build for this package. Here are the results. 

Buddy Build
- [ARM64 ](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=312960&view=results)

**Pull request merged**